spec/observables/from-promise-spec.js

```diff
@@ -1,6 +1,9 @@
 /* globals describe, it, expect */
 var Rx = require('../../dist/cjs/Rx');
 var Observable = Rx.Observable;
+var isNode = typeof process === 'object' &&
+             process + '' === '[object process]' &&
+             typeof window === 'undefined';
```

Comment:
(in other words, right there you're not *technically* testing if it's "Node", you're testing to see if it has `process`, which browsers shouldn't have, but I suppose they *could*. We deal with some pretty weird runtimes here at Netflix.